### ISSUE TITLE
Class 'Goetas\Xsd\XsdToPhp\Jms\Handler\BaseTypesHandler' not found

### ISSUE THREAD

**Issue by ericnicolaas** [AuthorizeNet/sdk-php#138]
I'm running into problems with being unable to pull in all the required dependencies. I have cloned the repo and run `composer update`. But when I attempt to make a payment, I hit this fatal error:
```` 
[23-May-2016 12:11:18 UTC] PHP Warning:  include(/vagrant/content/plugins/charitable-authorize-net/includes/libraries/anet_php_sdk/vendor/goetas/xsd2php/lib/Jms/Handler/BaseTypesHandler.php): failed to open stream: No such file or directory in /vagrant/content/plugins/charitable-authorize-net/includes/libraries/anet_php_sdk/autoload.php on line 16
[23-May-2016 12:11:18 UTC] PHP Warning:  include(): Failed opening '/vagrant/content/plugins/charitable-authorize-net/includes/libraries/anet_php_sdk/vendor/goetas/xsd2php/lib/Jms/Handler/BaseTypesHandler.php' for inclusion (include_path='.:/usr/share/php:/usr/share/pear') in /vagrant/content/plugins/charitable-authorize-net/includes/libraries/anet_php_sdk/autoload.php on line 16
[23-May-2016 12:11:18 UTC] PHP Fatal error:  Class 'Goetas\Xsd\XsdToPhp\Jms\Handler\BaseTypesHandler' not found in /vagrant/content/plugins/charitable-authorize-net/includes/libraries/anet_php_sdk/lib/net/authorize/api/controller/base/ApiOperationBase.php on line 82
````

Here is a screenshot of the `goetas/xsd2php/lib/Jms` directory. As you can see, there is no `Handler` folder. 

![screenshot 2016-05-24 08 57 18](https://cloud.githubusercontent.com/assets/676582/15487579/af37a9f6-218d-11e6-97b1-ece90d3d8038.png)

Is this working for others?

**Comment by ericnicolaas** [comment]
Just digging into the xsd2php repo and found that the `Handler` directory was removed as part of this commit: https://github.com/goetas/xsd2php/commit/db6c724ee4633968f16c5ca2d51bfcb7134fb925#diff-dab752bfae08ac54ebdeeec08972f84d

**Comment by asokol-teq** [comment]
hot-fix
```
"require-dev": {        
        "goetas/xsd2php": "2.0.x-dev#fdc2ab0bb3f2b3ab796ca567cf8c0f3446a7ea3a",
    },
```
works for me

**Comment by brianmc** [comment]
Thanks @asokol-teq  We're going to make the hotfix for right now but we will follow up with a move to the new goetas runtime xsd2php which will be a slimmer dependency for us.

Thanks again,

Brian

**Comment by brianmc** [comment]
We put in the temporary fix and the SDK and Sample Code builds are green again.  Thanks.

**Comment by LawanyaD** [comment]
If I run this code ("goetas/xsd2php": "2.0.x-dev#fdc2ab0bb3f2b3ab796ca567cf8c0f3446a7ea3a") the following error was occured, What to do???

Problem 1
    - The requested package goetas/xsd2php 2.0.x-dev#fdc2ab0bb3f2b3ab796ca567cf8c0f3446a7ea3a exists as goetas/xsd2php[1.0.0, 1.0.1, 2.0.0, 2.0.0-alpha, 2.1.0, dev-goetas-runtime, dev-master, 2.1.x-dev, dev-official-serializer] but these are rejected by your constraint.

**Comment by jhchnc** [comment]
Yes, what about the above question from Lawanya?

**Comment by akankaria** [comment]
Hi @LawanyaD & @jhchnc,
Appologies for the delay in response. 
Are you using latest PHP SDK 1.9. We have updated the dependency of goetas/xsd2php library in the latest release. 
If you are still facing the issue, please provide us with the steps you are following.

**Comment by davidreedernst** [comment]
My client's server is running PHP 5.3.x   The 1.8.8 authorizenet SDK would have worked (according to the composer.json file still displayed on http://developer.authorize.net/hello_world/ ) but because of the problem identified in this thread, it seems that there's no way to use PHP 5.3. The commit recommended by the hot fix says that it requires php 5.5. So, there's no way to run the authorizenet SDK on PHP 5.3 (or 5.4) anymore?

**Comment by edwinricaurte** [comment]
To skip the following error:

Problem 1
- The requested package goetas/xsd2php 2.0.x-dev#fdc2ab0bb3f2b3ab796ca567cf8c0f3446a7ea3a exists as goetas/xsd2php[1.0.0, 1.0.1, 2.0.0, 2.0.0-alpha, 2.1.0, dev-goetas-runtime, dev-master, 2.1.x-dev, dev-official-serializer] but these are rejected by your constraint.

Just add the following line to your composer.json file:
"minimum-stability": "dev"

Works for me.
